bin/dev.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ts-node
```

Comment:
why is dev.js `ts-node` but run.js `node`

Reply:
dev.js needs ts-node to be able to transpile the ESM src code at run time. Another option would be to use `node --loader=ts-node/esm`